mlir/lib/Dialect/QCO/Transforms/Mapping/Mapping.cpp

```
@@ -1258,32 +1215,15 @@ struct MappingPass : impl::MappingPassBase<MappingPass> {
     return success();
   }
 
-  std::shared_ptr<AugmentedDevice> device;
+  std::shared_ptr<SuperconductingDevice> device;
 };
 
 } // namespace
 
 std::unique_ptr<Pass>
-createMappingPass(const DenseSet<std::pair<size_t, size_t>>& couplingSet,
+createMappingPass(std::shared_ptr<SuperconductingDevice> device,
```

Cppcheck (2.21.0)

[style] The function 'createMappingPass' is never used.

(unusedFunction)